components/why-koda/Sections.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <section>
+    <div :class="containerClasses">
+      <div
+        v-for="section in mainSections"
+        :id="section.id"
+        :key="section.id"
+        class="py-14 md:py-[5.5rem] border-k-shade2"
+        :class="{
+          'border-b': section.id !== 'offline_exhibitions',
+          '!pb-0': section.id === 'onboarding',
+        }">
+        <WhyKodaSectionItem :section="section" />
+      </div>
+    </div>
+
+    <div>
+      <div class="flex gap-4 max-md:pb-14 max-md:pt-6 md:py-20 justify-center">
+        <div
+          v-for="img in offlineExhibitionsImages"
+          :key="img"
+          class="h-[216px] min-w-[291px] md:h-[358px] md:min-w-[480px] shadow-primary border rounded-xl overflow-hidden">
+          <BasicImage :src="img" alt="offline_exhibitions" />
+        </div>
+      </div>
+
+      <div :class="containerClasses">
+        <div class="pb-20">
+          <p class="text-xl md:text-2xl">{{ $t('benefits') }}:</p>
+          <div class="flex gap-4 flex-wrap !mt-4">
+            <div v-for="benefit in offlineExhibitionsBenefits" :key="benefit">
+              <span class="border rounded-full px-2 text-xl md:text-2xl">
+                {{ $t(`whyKoda.benefits.${benefit}`) }}
+              </span>
+            </div>
+          </div>
+        </div>
+
+        <div
+          v-if="lastSection"
+          :id="lastSection.id"
+          class="py-14 md:py-[5.5rem] border-k-shade2 border-t">
+          <WhyKodaSectionItem :section="lastSection" />
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+<script lang="ts" setup>
+const containerClasses = 'container mx-auto max-md:px-4'
```

Comment:
odd choice

why not put it in `style` block?

Reply:
fixed bb1eb2b, the `container`class was conflicting with tailwindcss